src/django/api/models.py

```diff
@@ -1742,3 +1742,74 @@ class FacilityLocation(models.Model):
     )
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class ApiLimit(models.Model):
+    """
+    Stores the number of requests a Contributor can make monthly.
+    """
+    contributor = models.ForeignKey(
```

Comment:
In our design notes we decided this should be a unique FK, but there is a system check warning suggesting that we change it to a `OneToOneField`. Consider making the suggested change. We already have a few of these, `Contributor.admin` for example

https://github.com/open-apparel-registry/open-apparel-registry/blob/6e2d71b190260edc975509611e86d0ed104627cf/src/django/api/models.py#L120-L124

If we to change the field type, we may need to regenerate the migration.

Reply:
Fixed in fb3fde8